README.md

```diff
@@ -74,24 +74,9 @@ CI runs a series of steps;  this the sequence to do it locally, along with some
 
 6. **Run the tests (without rubocop)**
 
-    Note that
-
-    ```
-    RAILS_ENV=test bundle exec rake ci
-    ```
-
-    is a shortcut for running the following 2 steps:
-
-    ```
-    RAILS_ENV=test bundle exec rake argo:repo:load
-    RAILS_ENV=test bundle exec rake spec
-    ```
-
-    `rake argo:repo:load` loads test fixture objects into fedora/solr.   You will NOT be able to re-run this task successfully unless fixtures are no longer in the docker containers (e.g. if you `docker-compose down`).  That is because they are already loaded.  The error messages in your terminal output do not surface this cause, but that is likely at play if you see `"Rubydora::FedoraInvalidRequest: See logger for details"`, esp with `"Caused by: RestClient::InternalServerError: 500 Internal Server Error"` in there. In this case, run only `bundle exec rake spec` instead.
-
-    Note that `RAILS_ENV=test` should not be necessary when running `bundle exec rake spec` on its own.
-
-    Note further that ```docker-compose down -v``` will wipe out the volumes for solr & fedora and allows you to start afresh (with ```docker-compose pull```) if your test data is in a strange state.
+```
+RAILS_ENV=test bundle exec rake ci
```

Comment:
Now that we no longer need to load fixtures, is the explicit `RAILS_ENV=test` still necessary? Or can we jettison that as well (here and in the CI configuration)?